dylint/src/library_packages/cargo_cli/util/hex.rs

```diff
@@ -11,16 +11,6 @@ use std::fs::File;
 use std::hash::{Hash, Hasher};
 use std::io::Read;
 
-pub fn to_hex(num: u64) -> String {
```

Comment:
Could I impose on you to undo the changes to this file, and add the following near the top, immediatley below `#![cfg_attr(dylint_lib = "supplementary", allow(unnamed_constant))]`?
```rs
#![cfg_attr(
    dylint_lib = "non_topologically_sorted_functions",
    allow(non_topologically_sorted_functions)
)]
```
The reason is that keeping this file as close to the original as possible will make it easier to check for differences.

Reply:
All of the changes to `library_packages/cargo_cli/util/hex.rs` except for the addition of that `#![cfg_attr(...)]`. That `#![cfg_attr(...)]` should prevent the lint from warning about that file's contents.